src/main/java/pages/SignUpPage.java

```diff
@@ -0,0 +1,42 @@
+package pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+public class SignUpPage {
+
+    WebDriver driver;
```

Comment:
no declaration fro webdriver here it should be extends from super class

Reply:
Done